README.md

```diff
@@ -769,6 +767,27 @@ there are basically two ways to do that:
 
 This will produce a single, executable, dead-code-eliminated file:
 
+**>= v0.15.0**
+
+Since v0.15.0 spago uses `esbuild` as the underlying default bundler. See the [`esbuild` getting started](https://esbuild.github.io/getting-started/#install-esbuild) for installation instructions.
+```bash
```

Comment:
```suggestion
```console
```

Reply:
I suppose all of the code blocks in this README use `bash`, but they ought to use `console` if they're indicating commands to run and printed results. Feel free to ignore this and we can update those later if you wish.

Reply:
Done.